src/logging_ffi.erl

```diff
@@ -12,26 +12,38 @@ configure() ->
         metadata => #{}
     }),
     logger:update_handler_config(default, #{
-        formatter => {logging_ffi, []}
+        formatter => {logging_ffi, #{colored => is_colored_output()}}
     }),
     nil.
 
-format(Event, _Config) ->
-    format(Event).
+is_colored_output() ->
+    case os:getenv("NO_COLOR") of
+        false -> true;
+        "false" -> true;
+        _ -> false
```

Comment:
Empty string is missing here!

Reply:
Hmm, I was considering empty string as the variable being set in my mind. 😅 But I understand your point.